thewalrus/fock_gradients.py

```diff
@@ -23,36 +23,72 @@
 .. autosummary::
     :toctree: api
 
-    Xgate
     Dgate
     Sgate
     Rgate
     Kgate
     S2gate
     BSgate
-    Sgate_real
-    S2gate_real
-    BSgate_real
 
 """
 import numpy as np
 
 from numba import jit
 
 
-from thewalrus.libwalrus import (
-    interferometer,
-    squeezing,
-    displacement,
-    interferometer_real,
-    displacement_real,
-    squeezing_real,
-    two_mode_squeezing,
-    two_mode_squeezing_real,
-)
+@jit(nopython=True)
+def displacement_rec(alpha, cutoff):  # pragma: no cover
```

Comment:
What does the `rec` mean? Ohhh, recursion?

Could just call this function `displacement` since there is no other displacement function it could get confused with?

Reply:
yeah. That can be done.